### PR TITLE
Fix config on startup

### PR DESCRIPTION
**Pull request by AppleDevelops**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [ ] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

## Pull Request description
...
This should fix the config error upon startup. I decided that I'd make a PR on this because I was tired of that error. It's probably not a great method but there it is.

## Changed Branch
...
Master

## Optional information
...

### CODE REVIEW

File: src/revivalpmmp/pureentities/PureEntities.php [hunk 1]
```diff
@@ -175,6 +175,7 @@ public function onLoad(){
 
     public function onEnable(){
         $this->getServer()->getPluginManager()->registerEvents(new EventListener($this), $this);
+	file_put_contents($this->getDataFolder() . "config.yml", $this->getResource("config.yml"));
```
Comment (kniffo80) [inline src/revivalpmmp/pureentities/PureEntities.php]: But this will override you settings each time you restart the server ...

**Comment by kniffo80** [comment]
I think the latest commit in https://github.com/RevivalPMMP/PureEntitiesX/pull/141 is a more reliable implementation. The branch fixes now finally the issue with writing default config to the disk.

**Comment by jasonwynn10** [comment]
This shouldn't be needed because of `SaveDefaultConfig()`

**Comment by AppleDevelops** [comment]
Yeah I realized

**Pull request closed without merging**